### PR TITLE
Feature clean up site

### PR DESCRIPTION
**Pull request by KrysOlechowski**
**Jira task:**
https://tipser.atlassian.net/secure/RapidBoard.jspa?rapidView=28&modal=detail&selectedIssue=T3-3332  (Aga is doing the same with tipser-elements-react-bootstrap)

**What?**
-cleaning up some elements, make it more readable.
-adding simple header with usefull links, and simple footer.

**Screens:**

BEFORE:
widget-old-img

AFTER:
widget-new-img

### CODE REVIEW

File: footer.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    font-size: 0.8em;
+    text-align: center;
+}
+.footer-link{
+   display: inline-block;
+}
+.footer-copyrights{
+   text-align: center;
+   margin-top:20px;
+}
```
Comment (sirpeas) [inline footer.css]: Empty line is needed at the end of every code file

File: header.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+   background-color: #fff;
+   min-height: 50px;
+   margin-bottom:10px;
+   display: flex;
+   flex-direction: column;
+   width: 100%;
+   justify-content: center;
+   position:relative;
+}
+.te-logo img{
```
Comment (sirpeas) [inline header.css]: add space before `{`

File: index.css [hunk 3]
```diff
@@ -5,39 +5,10 @@ body {
 
 .heading {
     text-align: center;
-    background: #f2f2f2;
-    padding: 20px 100px;
+    background-color: rgb(234, 230, 223);
```
Comment (sirpeas) [inline index.css]: Why not hex?

**Review by sirpeas — CHANGES_REQUESTED** [review]

File: footer.css (hunk 1 above)
Comment (KrysOlechowski) [reply footer.css]: done

File: header.css (hunk 2 above)
Comment (KrysOlechowski) [reply header.css]: done

File: index.css (hunk 3 above)
Comment (KrysOlechowski) [reply index.css]: done

File: index.html [hunk 4]
```diff
@@ -1,82 +1,112 @@
...
-                            console.error(error)
-                        },
-                        onAddToCart: function(options) {
-                            console.log('Hurray, you have added item to cart. ', options.product);
-                            console.log('Your cart size is now. ', options.cartSize);
-                        }
-            }
-                })
+                //The configuration of tipser widget comes here
+                env: "dev",
```
Comment (AgaK-O) [inline index.html]: According to the task `env` is supposed to be changed to `'prod'`.

**Comment by AgaK-O** [comment]
One small comment about the branch name. It should be formatted `feature/what-it-contains` and in this case, it's even better to use prefix `chore/...` as it is mostly cleanup, not a new feature.

File: index.html (hunk 4 above)
Comment (KrysOlechowski) [reply index.html]: done

**Comment by KrysOlechowski** [comment]
> One small comment about the branch name. It should be formatted `feature/what-it-contains` and in this case, it's even better to use prefix `chore/...` as it is mostly cleanup, not a new feature.

but adding header and footer is a feature?

**Pull request closed without merging**